src/hint.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+function getHint(secret, guess) {
+  if (guess.length < 4) {
+    return 'Please enter 4 digits!';
+  }
+
+  let bulls = 0;
+  const cows = [];
+  let copySecret = secret;
+
+  for (let i = 0; i < secret.length; i++) {
+    if (secret[i] === guess[i]) {
+      bulls++;
+      copySecret = secret.slice(i + 1, secret.length - i);
+    } else {
+      if (copySecret.includes(guess[i]) && !cows.includes(guess[i])) {
+        cows.push(guess[i]);
+      }
+    }
+  }
```

Comment:
It's a little bit hard to understand. Try to make it the following way:

Inside your `for` loop compare `guess[i]` and `secret[i]`. If they are equal increment `bulls`. Else, check if `secretArr.includes(guess[i])`. If it includes then increment `cows`.

Then return an object of bulls and cows (it's better to return an object with values because then you can use it to determine if the user has won).